CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## 1.2.0
+
+This release marks the first release under the prometheus-community organization.
+
+* [FEATURE] Added elasticsearch_clustersettings_stats_max_shards_per_node metric
```

Comment:
Usually we include `#XXX` for the PR that implements each feature. GitHub automatically links these, which is handy.

Reply:
Thanks. This should be fixed now.